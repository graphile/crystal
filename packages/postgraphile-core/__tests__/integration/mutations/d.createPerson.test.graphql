#> schemas: "d"

mutation {
  createPerson(input:{
    person:{
      lastName:"Doe"
      colNoOrder:"create1"
      colNoUpdate:"create2"
      colNoFilter:"create3"
    }
  }) {
    person{
      id
      firstName
      lastName
      colNoCreate
      colNoUpdate
      colNoOrder
      colNoFilter
      colNoCreateUpdate
      colNoCreateUpdateOrderFilter
    }
  }
}
