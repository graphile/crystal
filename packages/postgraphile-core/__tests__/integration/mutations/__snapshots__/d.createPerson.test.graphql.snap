// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`execute 1`] = `
Object {
  "data": Object {
    "createPerson": Object {
      "person": Object {
        "colNoCreate": "col_no_create",
        "colNoCreateUpdate": "col_no_create_update",
        "colNoCreateUpdateOrderFilter": "col_no_create_update_order_filter",
        "colNoFilter": "create3",
        "colNoOrder": "create1",
        "colNoUpdate": "create2",
        "firstName": null,
        "id": 10,
        "lastName": "Doe",
      },
    },
  },
}
`;
